specification/hugr.md

```diff
@@ -1147,6 +1163,14 @@ extensions:
     # outputs would be, in principle: Array<i+j>(t)
     # - but default type scheme interpreter does not support such addition
     # Hence, no signature block => will look up a compute_signature in registry.
+  - name: TupleConcat
+    description: "Concatenate two tuples"
+    params:
+      a: List[Type]
+      b: List[Type]
+    signature:
+      inputs: Sum([a]), Sum([b])
+      outputs: Sum([(a,b)]) # Syntax! Sum([a,b]) would be either-a-or-b, this has one variant
```

Comment:
Bleurgh. It'd be awkward to write these examples *not* in the "declarative format", but OTOH I don't think we are really moving in this direction, and it's very unclear how to write these ATM.

Reply:
I think better just to say what it is in the comment
```suggestion
      outputs: Sum([(a,b)]) # sum with a single variant of type (a, b)
```